coralproject/talk
# Enable webhooks for COMMENT_CREATED events

## What does this PR do?
This creates the ability to fire webhooks in response to new comments being created.

## What changes to the GraphQL/Database Schema does this PR introduce?
`COMMENT_CREATED` was added as a valid value for `WEBHOOK_EVENT_NAME`.

## How do I test this PR?
1. In the admin view, create or configure a webhook. Under "Events to send" you should now see `COMMENT_CREATED` as an option. 
2. Check this and save it. 
3. Create a comment and observe that a server listening for POST requests at the address specified in the webhook receives a POST request.
 
## How do we deploy this PR?
